boundary_layer/builders/templates/generator_operator.j2

```diff
@@ -29,23 +29,61 @@ def {{ item_name_builder }}(index, item):
  latter would discard any default task args, expecting them to be filled-in
  by airflow, while in fact airflow would not fill them in at all. #}
 {% set properties = node.resolved_properties.values %}
-for (index, item) in enumerate({{ iterable_builder }}(
+
+{% set all_items = (node.name + '_all_items') | sanitize_operator_name %}
+{{ all_items }} = {{ iterable_builder }}(
 {% for arg in builder_args %}
 {% if arg in properties %}
             {{ arg }} = {{ properties[arg] | format_value }},
 {% endif %}
 {% endfor %}
-        )):
+        )
+
+{% if node.batching_enabled %}
```

Comment:
Ok, so I did what you suggested, but I ended up adding a `batching_enabled` helper property to make it easier to resolve the batching situation.  Like you proposed, we can implicitly know that batching is enabled if batching settings exist (for now, that means there's a `batch_size`), and we can use the optional `disabled` property to turn batching off even when we have the batch size configured.  However, the simple check `if not node.batching.disabled` returns false positives here when the batching config does not exist at all, so the check needed to be more like `if node.batching|length > 0 and not node.batching.disabled`.  Since this is a bit cumbersome, I created the `batching_enabled` property to avoid having to duplicate this logic in multiple places (3 distinct places in this PR need to perform this check).